failures-summary-and-bottle-result/action.yml

```diff
@@ -35,7 +35,7 @@ runs:
         printf '### ${{ inputs.step_name }}\n' >> "$GITHUB_STEP_SUMMARY"
         "${{ inputs.collapse }}" && printf '\n<details><summary>Details</summary>\n<p>\n\n' >> "$GITHUB_STEP_SUMMARY"
         printf '```\n' >> "$GITHUB_STEP_SUMMARY"
-        tee -a "$GITHUB_STEP_SUMMARY" <"$full_result_path"
+        cat "$full_result_path" | sed 's/\x1B\[[0-9;]\{1,\}[A-Za-z]//g' | tee -a "$GITHUB_STEP_SUMMARY"
```

Comment:
```suggestion
        sed 's/\x1B\[[0-9;]\{1,\}[A-Za-z]//g' "$full_result_path" | tee -a "$GITHUB_STEP_SUMMARY"
```

Reply:
Might also be nicer to handle in `test-bot` before it dumps these files.

Reply:
@carlocab Thanks, makes sense. Yeh, I guess test-bot could do this stripping for us but feels like scoping it to GitHub Actions makes sense because it's a limitation of that.